community/kernel/src/main/java/org/neo4j/kernel/recovery/LatestCheckPointFinder.java

```diff
@@ -62,7 +62,7 @@ public LatestCheckPoint find( long fromVersionBackwards ) throws IOException
             LogVersionedStoreChannel channel = PhysicalLogFile.tryOpenForVersion( logFiles, fileSystem, version );
```

Comment:
For the future: Would be better if this actually threw the IOException and we caught only FileNotFoundException specifically.